applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp

```diff
@@ -819,18 +982,9 @@ namespace Kratos
         const ProcessInfo& rCurrentProcessInfo 
         )
     {
-        if ( rValues.size() != GetGeometry().IntegrationPoints(  ).size() )
-        {
-            rValues.resize( GetGeometry().IntegrationPoints(  ).size(), false );
-        }
-
-        for ( unsigned int ii = 0; ii < mConstitutiveLawVector.size(); ii++ )
-        {
-            rValues[ii] = mConstitutiveLawVector[ii]->GetValue( rVariable, rValues[ii] );
-        }
+        CalculateOnIntegrationPoints( rVariable, rValues, rCurrentProcessInfo );
```

Comment:
Certainty you can see other elements in other applications and the implementation is similar. Can you tell me which is the difference between the two methods?